server/src/scoring/validations/userValidate.ts

```diff
@@ -0,0 +1,11 @@
+﻿import { User } from '@prisma/client';
+import Joi from 'joi';
+
+const schema = Joi.object().keys({
+  nickname: Joi.string().allow(null, ''),
```

Comment:
Ja bym chyba po prostu zostawił to pole jako opcjonalne, wygodniej na froncie będzie nie dodać pola do obiektu niż ustawiać je explicit na null lub pusty string.

Reply:
.optional() === .allow(null, '') dziala tak samo, mozna wgl nie przekazywac tego z frontendu i tez nie ma bledow

Reply:
👍